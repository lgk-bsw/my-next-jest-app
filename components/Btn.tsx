import styles from "./Btn.module.scss";

export const Btn = ({ label }: { label: string; }) => (
    <button className={styles.btn}>
        {label}
    </button>
);