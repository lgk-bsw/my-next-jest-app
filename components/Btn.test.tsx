import React from "react";
import { Btn } from "./Btn";
import renderer from "react-test-renderer";

test("You can set label", () => {
    const component = renderer.create(
        <Btn label="My label" />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});