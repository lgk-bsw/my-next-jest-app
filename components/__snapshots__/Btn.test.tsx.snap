// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`You can set label 1`] = `
<button
  className="btn"
>
  My label
</button>
`;
